### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Florian Duruz

### DISCUSSION THREAD

**Discussion by Cat-du-Fromage** [heig-vd-dai-course/heig-vd-dai-course#248] [Show and tell]
https://github.com/Cat-du-Fromage/Cat-du-Fromage

**Comment by ludelafo** [comment]
Impressive project you're working on! I'm a big fan of RTS games myself (Age of Empires 3, Starcraft I/II, etc.). Is your project accessible somewhere? :)
